src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java

```diff
@@ -333,7 +334,7 @@ public void addNodeClinicalData(boolean header, boolean footer) {
                     //
                     ArrayList<ExportFormDataBean> forms = se.getExportFormData();
                     for (ExportFormDataBean form : forms) {
-                        if (!odmFilter.isCrossForm() || (odmFilter.isCrossForm() && !form.getStatus().equals(EventCRFStatus.INVALID.getI18nDescription(getLocale())))) {
+                        if (!odmFilter.isCrossForm() || (odmFilter.isCrossForm() )) {
```

Comment:
isn't this always going to be true?

Reply:
yes, I have to revisit this section later cause currently, we don't have Invalid status for EventCrf entity